Settings types error with v7.0.0-beta.2

Likely the same as #8646

```
node_modules/@pixi/settings/index.d.ts:3:23 - error TS2688: Cannot find type definition file for 'css-font-loading-module'.

3 /// <reference types="css-font-loading-module" />
```

API Extractor like to inline the `css-font-loading-module` types.

Settings now has a reference to `FontFaceSet` 

https://github.com/pixijs/pixijs/blob/b82316094043a2a45a0c305068e5ebe8321f668a/packages/settings/src/adapter.ts#L19

Was added in #8698.

--- a/packages/settings/package.json
+++ b/packages/settings/package.json
@@ -36,7 +36,8 @@
     "*.d.ts"
   ],
   "dependencies": {
-    "@pixi/constants": "7.0.0-beta.2"
+    "@pixi/constants": "7.0.0-beta.2",
+    "@types/css-font-loading-module": "^0.0.7"
   },
   "devDependencies": {
     "ismobilejs": "^1.1.0"

